fee_allocator/accounting/distribution.py

```diff
@@ -112,66 +101,36 @@ def re_distribute_incentives(
-            bal_incentives_to_increase = round(
-                _total_incentives * (aura_incentives_pct - aura_vebal_share), 2)
-            # Calculate how much we need to decrease aura incentives but not to go below min aura incentive
-            if _aura_incentives - bal_incentives_to_increase <= min_aura_incentive:
-                # If aura incentives are less than min aura incentive after decreasing, we need to calculate
-                # how much we need to decrease aura incentives to min aura incentive
-                aura_incentives_to_decrease = _aura_incentives - min_aura_incentive
-                # Decrease aura incentives to min aura incentive
-                incentives[pool_id]['aura_incentives'] -= aura_incentives_to_decrease
-                incentives[pool_id]['bal_incentives'] += aura_incentives_to_decrease
-            else:
-                # Increase bal incentives
-                incentives[pool_id]['bal_incentives'] += bal_incentives_to_increase
-                # Decrease aura incentives
-                incentives[pool_id]['aura_incentives'] -= bal_incentives_to_increase
-        # If aura incentives percentage is more than aura_vebal_share, we need to increase aura incentives
-        else:
-            # Calculate how much we need to increase aura incentives
-            aura_incentives_to_increase = round(
-                _total_incentives * (aura_vebal_share - aura_incentives_pct), 2)
-            # Only increase aura incentives if it's more than min aura incentive
-            if _aura_incentives + aura_incentives_to_increase < min_aura_incentive:
-                continue
-            # Increase aura incentives
-            incentives[pool_id]['aura_incentives'] += aura_incentives_to_increase
-            # Decrease bal incentives
-            incentives[pool_id]['bal_incentives'] -= aura_incentives_to_increase
+        if _data['aura_incentives'] < min_aura_incentive:
+            # Calculate incentives to redistribute
+            incentives_to_redistribute = _data['aura_incentives']
+            # Set incentives to redistribute to 0
+            incentives[pool_id]['aura_incentives'] = 0
+            incentives[pool_id]['bal_incentives'] += incentives_to_redistribute
+            debt_to_aura_market += incentives_to_redistribute
+
+    if debt_to_aura_market:
+        amount_per_pool =  round(debt_to_aura_market / 6, 4)
+        # Create an array of the top 6 pools that received the most total_incentives
+        top_6_pools = sorted(incentives.items(), key=lambda x: x[1]['total_incentives'], reverse=True)[:6]
+        # Distribute  1/6th of incentives_taken_from_aura_market to each of the top 6 pools, while subtracting the same amount from bal_incentives
+        for pool_id, _data in top_6_pools:
+            if _data['total_incentives'] > 0:
+                # TODO:  Still an edge case here.  If amount per pool is not allocated, we'll still underallocate to
```

Comment:
If the pool doesn't have enough incentives to move, we move all it's got.  Aura will suffer a little today until we figure out better code.  This seems unlikely with the top 6 earning pools.  We can also increase or decrease how many pools to consider, so long as we are sure that they will all be getting some aura votes already.

Reply:
For now this is ok. For next iteration make the number of pools a config parameter  if we decide to go down this path